src/Browser.php

```diff
@@ -64,6 +66,7 @@ public function __construct(LoopInterface $loop, ConnectorInterface $connector =
             $this->messageFactory,
             $loop
         );
+        $this->setCookieJar(new CookieJar());
```

Comment:
Not sure about a default here, but maybe not using a cookie jar by default would alleviate privacy concern and reduce BC impact?